src/wpt/viewer/index.html

```diff
@@ -61,6 +66,13 @@
 			fill: CanvasText;
 			font-family: "GuardianTextSans";
 		}
+
+		@media only screen and (min-width: 960px) {
+
+			.performance td {
+				font-size: 5rem;
+			}
```

Comment:
Instead of a media query, maybe this should be a viewport size? 

`clamp(2rem, 3vw, 5rem)`

https://developer.mozilla.org/en-US/docs/Web/CSS/clamp